src/services/hotjar/HotjarFactory.ts

```diff
@@ -0,0 +1,14 @@
+import HotjarService from 'src/services/hotjar/HotjarService';
```

Comment:
Could this live in AnalyticsFactory? `AnalyticsFactory.hotjar()` Unless you think analytics and hot jar are two different concerns.

Reply:
It makes sense, will move it to analytics